ThreadData.ts

```diff
@@ -122,19 +122,19 @@ export class ThreadData {
     }
 
     validateActions() {
-        if (!this.thread_action.hasAnyAction()) {
+        if (!this.thread_action.hasAnyAction() && this.thread_action.move_to != InboxActionType.NOTHING) {
```

Comment:
I think this addition does nothing. We just checked that `hasAnyAction() == False`, which is:
```
    hasAnyAction() {
        return this.label_names.size > 0
            || this.move_to != InboxActionType.DEFAULT
            || this.important != BooleanActionType.DEFAULT
            || this.read != BooleanActionType.DEFAULT;
    }
```
So we know that `this.move_to == InboxActionType.DEFAULT`. So the `&& move_to != NOTHING` will always be true.